src/MessageManager.cpp

```diff
@@ -207,7 +215,7 @@ void MessageManager::Unsubscribe( IMessageSubscriber* pSubscriber, MessageID m )
 void MessageManager::Broadcast( Message &msg ) const
 {
 	// GAMESTATE is created before MESSAGEMAN, and has several BroadcastOnChangePtr members, so they all broadcast when they're initialized.
-	if(m_Logging)
+	if(this != nullptr && m_Logging)
```

Comment:
A different PR can change MessageManager from a class to a namespace.  The singleton construction order forces MessageManager to not have any real member variables, so it should be a namespace to prevent future problems.